examples/calc_spectrumkurucz.py

```diff
@@ -0,0 +1,56 @@
+from radis.lbl.factory import SpectrumFactory
```

Comment:
I see you already have a plot_kuruzc. Choose and keep only one example

Reply:
The plot_kurucz example was created without using Spectrum Factory and is based on the first method named process in the Kurucz API. The one that I will keep is the calc_spectrum example because it uses SpectrumFactory. Thank you for the comment, I will rename it so that it appears in the galley